bridges/otelzerolog/hook_test.go

```diff
@@ -3,13 +3,18 @@
 package otelzerolog
 
 import (
+	"os"
+	// "reflect".
```

Comment:
Why keep those as comments? (same for `time`). If we're not using those, we should remove them.

Reply:
oh, that was an error, was trying out some things 
Will take it out now